connectors/easy-post/element-templates/easy-post-connector.json

```diff
@@ -0,0 +1,943 @@
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.parcel.predefined_package"
+      },
+      "constraints": {
+        "notEmpty": false
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createParcel"
+        ]
+      }
+    },
+    {
+      "label": "To address",
+      "description": "ID of the destination address",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.shipment.to_address.id"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createShipment"
+        ]
+      }
+    },
+    {
+      "label": "From address",
+      "description": "ID of the origin address",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.shipment.from_address.id"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createShipment"
+        ]
+      }
+    },
+    {
+      "label": "Parcel",
+      "description": "ID of the parcel",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.shipment.parcel.id"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createShipment"
+        ]
+      }
+    },
+    {
+      "label": "Shipment id",
+      "description": "ID of the shipment",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "shipmentIdValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "buyShipment"
+        ]
+      }
+    },
+    {
+      "label": "Rate ID",
+      "description": "ID of the actual rate quote for this service",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.rate.id"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "buyShipment"
+        ]
+      }
+    },
+    {
+      "label": "Address ID",
+      "description": "ID of the salready created addres",
```

Comment:
typo and please replace with: The ID of the address that has already been created.